src/components.rs

```diff
@@ -50,6 +56,8 @@ pub enum ParticleSpace {
 /// If a [`ParticleSystem`] component is removed before all particles have finished their lifetime, the associated particles will all despawn themselves
 /// on the next frame.
 #[derive(Debug, Component, Clone)]
+#[cfg_attr(feature = "bevy_reflect", derive(Reflect))]
+#[cfg_attr(feature = "bevy_reflect", reflect(Component))]
```

Comment:
Should we add `Reflect` to the other components that go with the particle system entity as well?

- `ParticleCount`
- `RunningState`
- `BurstIndex`